examples/hacktoberfest-diagram/hacktoberfest-diagram.js

```diff
@@ -0,0 +1,406 @@
+    updateCellLabel(bpmnVisualizationHacktoberfestLightTheme, cellId, value);
+    updateCellLabel(bpmnVisualizationHacktoberfestDarkTheme, cellId, value);
+}
+
+inputProjectName.oninput = function(event) {
+    let projectName = event.target.value;
+    bpmn = bpmnDiagram(projectName);
+
+    updateCellsLabel("call_activity", `Contribute to ${projectName} 🔧`);
+    updateCellsLabel( "user_task_5", `Tweet how it was fun to contribute to ${projectName} 😃`);
+};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// BPMN
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+function bpmnDiagram(projectName) {
+    return  `<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="definition_1" >
+  <bpmn:collaboration id="collaboration_1">
+    <bpmn:participant id="participant_1" processRef="process_1" name="Hacktoberfest participation"/>
+  </bpmn:collaboration>
+  <bpmn:process id="process_1" isExecutable="false">
+    <bpmn:startEvent id="start_event" name="Hacktoberfest starts on October, 1st">
+      <bpmn:outgoing>sequence_flow_1</bpmn:outgoing>
+      <bpmn:timerEventDefinition />
+    </bpmn:startEvent>
+    <bpmn:exclusiveGateway id="exclusive_gateway_1" name="Already registered on Hacktoberfest website ?" default="sequence_flow_2">
+      <bpmn:incoming>sequence_flow_1</bpmn:incoming>
+      <bpmn:outgoing>sequence_flow_2</bpmn:outgoing>
+      <bpmn:outgoing>sequence_flow_4</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:userTask id="user_task_1" name="Register on https://hacktoberfest.&#10;digitalocean.com">
+      <bpmn:incoming>sequence_flow_2</bpmn:incoming>
+      <bpmn:outgoing>sequence_flow_3</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:exclusiveGateway id="exclusive_gateway_2">
+      <bpmn:incoming>sequence_flow_3</bpmn:incoming>
+      <bpmn:incoming>sequence_flow_4</bpmn:incoming>
+      <bpmn:outgoing>sequence_flow_5</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:userTask id="user_task_2" name="Find an issue labeled &#39;hacktoberfest&#39; 🔍">
+      <bpmn:incoming>sequence_flow_5</bpmn:incoming>
+      <bpmn:incoming>sequence_flow_8</bpmn:incoming>
+      <bpmn:outgoing>sequence_flow_6</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:callActivity id="call_activity" name="Contribute to ${projectName} 🔧">
+      <bpmn:incoming>sequence_flow_6</bpmn:incoming>
+      <bpmn:outgoing>sequence_flow_7</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:exclusiveGateway id="exclusive_gateway_3" name="Is it your your 5th contribution?" default="sequence_flow_8">
```

Comment:
In 2020, only 4 pull requests are required (https://hacktoberfest.digitalocean.com/details/#rules)
> To get a shirt, you must make four pull requests (PRs) between October 1–31 in any time zone.